server/test/handlers/getPossibleTimeslots.test.js

```diff
@@ -0,0 +1,140 @@
+            },
+            {
+                slot_id: 4,
+                start_time: '09:30:00',
+                end_time: '10:00:00'
+            },
+            {
+                slot_id: 5,
+                start_time: '10:00:00',
+                end_time: '10:30:00'
+            },
+            {
+                slot_id: 6,
+                start_time: '10:30:00',
+                end_time: '11:00:00'
+            },
+            {
+                slot_id: 7,
+                start_time: '11:00:00',
+                end_time: '11:30:00'
+            },
+            {
+                slot_id: 8,
+                start_time: '11:30:00',
+                end_time: '12:00:00'
+            },
+            {
+                slot_id: 9,
+                start_time: '12:00:00',
+                end_time: '12:30:00'
+            },
+            {
+                slot_id: 10,
+                start_time: '12:30:00',
+                end_time: '13:00:00'
+            },
+            {
+                slot_id: 11,
+                start_time: '13:00:00',
+                end_time: '13:30:00'
+            },
+            {
+                slot_id: 12,
+                start_time: '13:30:00',
+                end_time: '14:00:00'
+            },
+            {
+                slot_id: 13,
+                start_time: '14:00:00',
+                end_time: '14:30:00'
+            },
+            {
+                slot_id: 14,
+                start_time: '14:30:00',
+                end_time: '15:00:00'
+            },
+            {
+                slot_id: 15,
+                start_time: '15:00:00',
+                end_time: '15:30:00'
+            },
+        ];
+
+        // Act
+        const possibleTimeslots = await timeslotHandler.getPossibleTimeslots();
+
+        // Assert
+        expect(possibleTimeslots).toMatchObject(testTimeslotDetails);
+    });
+
+
+    test('testing specified time parameters expecting a return', async () => {
+        // Arrange
+        const testTimeslotDetails = [
+            {
+                slot_id: 5,
+                start_time: '10:00:00',
+                end_time: '10:30:00'
+            },
+            {
+                slot_id: 6,
+                start_time: '10:30:00',
+                end_time: '11:00:00'
+            },
+            {
+                slot_id: 7,
+                start_time: '11:00:00',
+                end_time: '11:30:00'
+            },
+            {
+                slot_id: 8,
+                start_time: '11:30:00',
+                end_time: '12:00:00'
+            },
+        ];
+
+        // Act
+        const possibleTimeslots = await timeslotHandler.getPossibleTimeslots('10:00:00', '12:00:00');
+
+        // Assert
+        expect(possibleTimeslots).toMatchObject(testTimeslotDetails);
+    });
+
+    test('testing specified time parameters expecting null return ', async () => {
+        // Arrange
+        const testTimeslotDetails = [
```

Comment:
Hi Peter,
To reduce the number of code lines, I would suggest combining lines 130 and 131 to show [] rather than having the brackets spread across two lines.

Reply:
Resolved!